mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactLoggedModelView.tsx

```diff
@@ -207,6 +208,42 @@ validate_serving_input(model_uri, serving_payload)`;
     }
   }
 
+  validateModelPredictText(modelPath: any, inputExample?: string) {
```

Comment:
Is `inputExample` always a valid python code? Here's an example:

```json
{ "a": true }
```

Reply:
Ah good point :)

Reply:
Updated to use json.loads!